terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/ApServerName.java

```diff
@@ -0,0 +1,12 @@
+package org.terasoluna.gfw.functionaltest.app;
+
+/**
+ * Enumeration class for identifying application server.
+ * <p>
+ * If application server name is not set in application.env.properties, <br/>

```

Comment:
OK. But is new line necessary? Please check format using `formatter-maven-plugin`.